File: src/main/Java/boj_step8/boj2720.java
import java.io.*;

public class boj2720 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int t = Integer.parseInt(reader.readLine());
        while (t-- > 0) {
            int c = Integer.parseInt(reader.readLine());
            writer.append(c / 25 + " ");
            c %= 25;
            writer.append(c / 10 + " ");
            c %= 10;
            writer.append(c / 5 + " ");
            c %= 5;
            writer.append(String.valueOf(c));
            writer.newLine();
        }
        writer.close();
    }
}

File: src/main/kotlin/others/boj2720.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val test = r.readLine().toInt()
    val (quarter, dime, nickel, penny) = "25 10 5 1".split(" ").map { it.toInt() }
    repeat(test) {
        var change = r.readLine().toInt()
        w.write("${change / quarter} ")
        change %= quarter
        w.write("${change / dime} ")
        change %= dime
        w.write("${change / nickel} ")
        change %= nickel
        w.write("${change}\n")
    }
    w.close()
}